Failed to import MatBasicSpinnerDirective in Angular v18.

Cannot import MatBasicSpinnerDirective or NgxLoadingButtonsModule with Angular v18 standalone.

```bash
error TS2307: Cannot find module 'ngx-loading-buttons' or its corresponding type declarations.
```

```json
"ngx-loading-buttons": "^18.0.0"
"typescript": "^5.4.2"
"@angular/core": "^18.0.1",
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -28,7 +28,7 @@ jobs:
 
       - uses: actions/setup-node@v4
         with:
-          node-version: latest
+          node-version: lts/*
 
       - name: Install dependencies
         # If checksum for the `package-lock.json` hasn't changed then we shouldn't run

--- a/.github/workflows/npmpublish.yml
+++ b/.github/workflows/npmpublish.yml
@@ -1,7 +1,7 @@
 name: NPM Publish
 
 env:
-  NODE_VERSION: latest
+  NODE_VERSION: lts/*
 
 on:
   release:

--- a/projects/ngx-loading-buttons/package.json
+++ b/projects/ngx-loading-buttons/package.json
@@ -22,9 +22,6 @@
   "repository": {
     "url": "https://github.com/dkreider/ngx-loading-buttons"
   },
-  "files": [
-    "styles.css"
-  ],
   "exports": {
     "./styles": "./styles.css",
     "./styles.css": "./styles.css"

--- a/projects/ngx-loading-buttons/package.json
+++ b/projects/ngx-loading-buttons/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ngx-loading-buttons",
-  "version": "18.0.0",
+  "version": "18.0.1",
   "peerDependencies": {
     "@angular/common": "^18.0.0",
     "@angular/core": "^18.0.0",

--- a/projects/ngx-loading-buttons/package.json
+++ b/projects/ngx-loading-buttons/package.json
@@ -23,7 +23,7 @@
     "url": "https://github.com/dkreider/ngx-loading-buttons"
   },
   "exports": {
-    "./styles": "./styles.css",
-    "./styles.css": "./styles.css"
+    "./styles": "./src/styles.css",
+    "./styles.css": "./src/styles.css"
   }
 }

